test/cpp/interop/xds_interop_server.cc

```diff
@@ -94,12 +161,18 @@ int main(int argc, char** argv) {
     std::cout << "Failed to get hostname, terminating" << std::endl;
     return 1;
   }
+  int port = absl::GetFlag(FLAGS_port);
   if (absl::GetFlag(FLAGS_port) == 0) {
     std::cout << "Invalid port, terminating" << std::endl;
     return 1;
   }
-
-  RunServer(absl::GetFlag(FLAGS_port), hostname);
+  int maintenance_port = absl::GetFlag(FLAGS_maintenance_port);
+  if (absl::GetFlag(FLAGS_port) == 0) {
```

Comment:
use port above

Reply:
actually you mean maintenance_port right?

Reply:
Right, this should be `maintenance_port`